templates/master.yaml.tmpl

```diff
@@ -3284,7 +3284,7 @@ systemd:
       --register-node=true \
       --register-with-taints=node-role.kubernetes.io/master=:NoSchedule \
       --kubeconfig=/etc/kubernetes/kubeconfig/kubelet.yaml \
-      --node-labels="node-role.kubernetes.io/master,role=master,ip=${DEFAULT_IPV4},master-id={{.MasterID}}" \
```

Comment:
Before merging / propagating this, I think we need to take care of current uses of `role=master`. I for example just created one node selector based on this for tenant last week (and yes, I understand this is for CP, but consistency).